src/js/components/__tests__/ServiceDetailVolumesTab-test.js

```diff
@@ -1,56 +0,0 @@
-jest.dontMock('../../mixins/GetSetMixin');
```

Comment:
I removed this file in favor of integration tests because it ended up being just a simple table.